impl/src/main/java/io/jsonwebtoken/impl/DefaultJwtBuilder.java

```diff
@@ -296,9 +296,8 @@ public String compact() {
 
         if (this.serializer == null) {
             //try to find one based on the runtime environment:
-            InstanceLocator<Serializer<Map<String,?>>> locator =
```

Comment:
@lhazlewood Any idea why this was using reflection for a class in the same module?  Maybe an artifact of some older refactor?

Reply:
I believe this was old cruft that wasn't cleaned up when we made the `api` and `impl` modules.  The new change looks correct.